src/gimelstudio/core/output_node.py

```diff
@@ -38,12 +40,33 @@ def NodeMeta(self):
         }
         return meta_info
 
+    def NodeInitProps(self):
+        self.export_button = api.ActionProp(
+            idname="export",
+            label="Export",
+            action=self.OnExportButtonPressed
+        )
+        self.NodeAddProp(self.export_button)
+
     def NodeInitParams(self):
         p = api.RenderImageParam('Image', 'Image')
         self.NodeAddParam(p)
 
     def NodeEvaluation(self, eval_info):
         pass
 
+    def OnExportButtonPressed(self, event):
+        # TODO: This is messy. Need to find a cleaner solution to accessing the renderer
+        app_frame = self.nodegraph.parent.parent
```

Comment:
I agree, this is messy. 😃  Though, I am not sure how exactly we can fix it. Even moving this to a callable method in ``node.py`` only "hides" some of the mess. Any ideas?